packages/engine/bin/cli/src/main.rs

```diff
@@ -107,6 +110,8 @@ async fn main() -> Result<()> {
 
     let _guard = hash_engine::init_logger(
         args.emit,
+        OutputLocation::default(),
+        "./log",
```

Comment:
If we're already making configurable for one binary, lets just add it to the CLI args

Reply:
Updated the CLI with those arguments as well